core/server/worker/src/main/java/alluxio/worker/block/SpaceReserver.java

```diff
@@ -77,19 +77,23 @@ public SpaceReserver(BlockWorker blockWorker) {
         // High watermark defines when to start the space reserving process
         PropertyKey tierHighWatermarkProp =
             PropertyKey.Template.WORKER_TIERED_STORE_LEVEL_HIGH_WATERMARK_RATIO.format(ordinal);
-        Preconditions.checkArgument(Configuration.getDouble(tierHighWatermarkProp) > 0,
-            "The high watermark of tier " + ordinal + " should be positive");
-        Preconditions.checkArgument(Configuration.getDouble(tierHighWatermarkProp) < 1,
-            "The high watermark of tier " + ordinal + " should be less than 1.0");
-        long highWatermark =
-            (long) (tierCapacity * Configuration.getDouble(tierHighWatermarkProp));
+        double tierHighWatermarkConf = Configuration.getDouble(tierHighWatermarkProp);
+        Preconditions.checkArgument(tierHighWatermarkConf > 0, "The high watermark of tier "
+            + ordinal + " should be positive, but is " + tierHighWatermarkConf);
+        Preconditions.checkArgument(tierHighWatermarkConf < 1, "The high watermark of tier "
+            + ordinal + " should be less than 1.0, but is " + tierHighWatermarkConf);
+        long highWatermark = (long) (tierCapacity * Configuration.getDouble(tierHighWatermarkProp));
         mHighWatermarks.put(tierAlias, highWatermark);
 
         // Low watermark defines when to stop the space reserving process if started
         PropertyKey tierLowWatermarkProp =
             PropertyKey.Template.WORKER_TIERED_STORE_LEVEL_LOW_WATERMARK_RATIO.format(ordinal);
-        Preconditions.checkArgument(Configuration.getDouble(tierLowWatermarkProp) >= 0,
-            "The low watermark of tier " + ordinal + " should not be negative");
+        double tierLowWatermarkConf = Configuration.getDouble(tierLowWatermarkProp);
+        Preconditions.checkArgument(tierLowWatermarkConf >= 0, "The low watermark of tier "
```

Comment:
no string concatenation please for `checkArgument`, 
use something like `checkArgument(i < j, "Expected i < j, but %s > %s", i, j);` to avoid expensive string construction